common/src/main/java/feast/common/function/Store.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.common.function;
+
+import feast.proto.core.StoreProto;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class Store {
+
+  /**
+   * Accepts a comma-delimited Subscriptions that is string-formatted and converts it to a list of
+   * Subscription class objects.
+   *
+   * @param subscriptions String formatted Subscriptions, comma delimited.
+   * @param exclude flag to determine if subscriptions with exclusion flag should be returned
+   * @return List of Subscription class objects
+   */
+  public static List<StoreProto.Store.Subscription> parseSubscriptionFrom(
+      String subscriptions, boolean exclude) {
+    return Arrays.stream(subscriptions.split(","))
+        .map(subscriptionStr -> convertStringToSubscription(subscriptionStr, exclude))
+        .collect(Collectors.toList());
+  }
+
+  /**
+   * Accepts a Subscription class object and returns it in string format
+   *
+   * @param sub Subscription class to be converted to string format
+   * @return String formatted Subscription class
+   */
+  public static String parseSubscriptionFrom(StoreProto.Store.Subscription sub) {
+    if (sub.getName().isEmpty() || sub.getProject().isEmpty()) {
+      throw new IllegalArgumentException(
+          String.format("Missing arguments in subscription string: %s", sub.toString()));
+    }
+
+    return String.format("%s:%s:%s", sub.getProject(), sub.getName(), sub.getExclude());
+  }
+
+  /**
+   * Accepts a exclude parameter to determine whether to return subscriptions that are excluded.
+   *
+   * @param sub String formatted Subscription to be converted to Subscription class
+   * @param exclude flag to determine if subscriptions with exclusion flag should be returned
```

Comment:
Why is it the responsibility of this method to have this `excluded` flag? Why cant the caller filter the list?